### ISSUE TITLE
'requiredError' Common Field Property

### ISSUE THREAD

**Issue by kevin-howard-jd** [solid/solid-ui#138]
## User Story
As a Form Designer, I can define the error to be displayed if a user fails to enter a required field, so the fact that the field is required is clear to the user.

## Acceptance Criteria
1. In a Form Model, the 'requiredError' property can be optionally defined for a data field, that defines the error message that should be displayed is the user fails to enter a value.
2. When the form is rendered, if the data field is required and the user fails to enter a value, the defined 'requiredError' error message is displayed.

**Comment by timbl** [comment]
Friendly amendment:  `requiredErrorMessage` If you want a message...

 or maybe we instantiate the error itself as a thing, and then have many labels on in different languages?

```
   ui: requiredError [ rdfs:label "You have to give your SSN, sorry"@en ]
```

**Comment by kevin-howard-jd** [comment]
Agreed on the amendment to the property name. I will make the adjustment to the User Story.

**Comment by james-martin-jd** [comment]
@timbl One question I have is why use rdfs:label? In other form language examples I've seen a ui:label used instead, and I think they could have different semantic meanings (e.g. a label specifically designed and formatted to be consumed by a UI). If so, we could add a new issue for ui:label to be added, or alternately switch to rdfs:label for consistency.
